mesonbuild/cmake/fileapi.py

```diff
@@ -257,6 +256,27 @@ def parse_project(pro: T.Dict[str, T.Any]) -> T.Dict[str, T.Any]:
 
             return pro_data
 
+        def find_target(cnf_data: T.Dict[str, T.Any], id: str) -> T.Optional[T.Dict[str, T.Any]]:
```

Comment:
Since these functions aren't exported, I would prefix them with an underscore

Reply:
done!

Reply:
Perfect

Reply:
NACK, please revert. It's an unnecessary change and the point of underscores is to mark private-but-exported functions as "should not be used" and "is not imported by star imports".

Functions defined inside of other functions get unbound when the function completes and do not need underscores.

Reply:
My bad, I saw this indented, and didn't see it was just within a function.

Reply:
No problem, reverted!

Reply:
It can be confusing to spot the difference, my trick is to check whether the first argument is `self` -- functions that are aligned at the start of the line are usually module functions, and functions that take `self` are usually class methods, functions that do neither one are usually temporary functions.